viper/modules/virustotal.py

```diff
@@ -355,11 +368,17 @@ def run(self):
                 if eid:
                     if __sessions__.is_attached_misp(quiet=True):
                         if __sessions__.current.misp_event.event.id != int(eid):
-                            self.log('warning', 'You opened a sample related to a MISP event different than the one you are currently connected to: {}.'.format(eid))
+                            self.log('warning',
+                                     'You opened a sample related to a MISP event different than the one you are '
+                                     'currently connected to: {}.'.format(
+                                         eid))
                         else:
                             self.log('success', 'You opened a sample related to the current MISP event.')
                     else:
-                        self.log('warning', 'This samples is linked to the MISP event {eid}. You may want to run misp pull {eid}'.format(eid=eid))
+                        self.log('warning',
```

Comment:
Yes, I'm cool with 100/120, but only f it doesn't get in the way of reading the code.